e2e_tests/src/e2e_conduit/scenarios/__init__.py

```diff
@@ -9,12 +9,20 @@
 @dataclass
 class Scenario:
     """Data class for conduit E2E test pipelines"""
+
     name: str
     importer: ImporterPlugin
     processors: list[PluginFixture]
     exporter: PluginFixture
     accumulated_config: dict = field(default_factory=dict)
     conduit_dir: str = ""
 
+    def get_validation_errors(self) -> list[str]:
```

Comment:
we actually run accounting validation in indexer e2e. we should probably create a ticket to migrate that to conduit.

Reply:
Thanks for the suggestion. I created issue #123